### PR TITLE
feat(feedback): process feedbacks in ingestion

### PR DESCRIPTION
**Pull request by JoshFerge**
- Handles feedback ingestion for feedbacks that come in as events.

### CODE REVIEW

File: src/sentry/ingest/consumer/processors.py [hunk 1]
```diff
@@ -145,6 +144,14 @@ def process_event(message: IngestMessage, project: Project) -> None:
             event_id=event_id,
             project_id=project_id,
         )
+    elif data.get("type") == "feedback":
+        save_event_feedback.delay(
```
Comment (JoshFerge) [inline src/sentry/ingest/consumer/processors.py:148]: let's add the feature flag check here

**Comment by JoshFerge** [comment]
there are only a few lines of code added and in the test we mock one of them, which is tested downstream. i think this is O.K.

File: tests/sentry/ingest/ingest_consumer/test_ingest_consumer_processing.py [hunk 2]
```diff
@@ -138,6 +148,51 @@ def test_transactions_spawn_save_event_transaction(
...
+            "start_time": start_time,
+            "event_id": event_id,
+            "project_id": project_id,
+            "remote_addr": "127.0.0.1",
+        },
+        project=default_project,
+    )
+    assert not len(preprocess_event)
+    assert save_event_feedback.delay.call_args[0] == ()
+    assert (
```
Comment (bruno-garcia) [inline tests/sentry/ingest/ingest_consumer/test_ingest_consumer_processing.py:190]: should we assert `event["type"]` too here?

**Review by bruno-garcia — APPROVED** [review]
I'm not familiar with this part of the code but LGTM

File: src/sentry/ingest/consumer/processors.py [hunk 3]
```diff
@@ -145,6 +144,15 @@ def process_event(message: IngestMessage, project: Project) -> None:
             event_id=event_id,
             project_id=project_id,
         )
+    elif data.get("type") == "feedback":
+        if features.has("organizations:user-feedback-ingest", project.organization, actor=None):
+            save_event_feedback.delay(
+                cache_key=None,  # no need to cache as volume is low
```
[the remark this answers is not in the document]
Comment (JoshFerge) [reply src/sentry/ingest/consumer/processors.py:150]: https://github.com/getsentry/sentry/blob/33ed7d6ab179f1dd3e2a102e0d22badd7f140181/src/sentry/tasks/store.py#L874

cache_key is currently not use in the callee's function.

**Pull request merged**